iina/Base.lproj/MainMenu.xib

```diff
@@ -264,12 +264,12 @@
                             <menuItem title="Volume:" enabled="NO" id="iQF-1S-9EK">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                             </menuItem>
-                            <menuItem title="Volume + 5%" tag="10" keyEquivalent="" id="Wbq-vt-hzX"/>
-                            <menuItem title="Volume + 1%" tag="1" alternate="YES" keyEquivalent="" id="c6Y-0e-vgf">
+                            <menuItem title="Increase Volume 5%" tag="10" keyEquivalent="" id="Wbq-vt-hzX"/>
+                            <menuItem title="Increase Volume 1%" tag="1" alternate="YES" keyEquivalent="" id="c6Y-0e-vgf">
```

Comment:
I think "+" and "-" are more intuitive.

(Shouldn't it be "Increase Volume _by_ 5%"? All other localizations should also be adapted for this change BTW)